Common/TDSQASystemAPI/TestPackage/PresentationsPresentation.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Generic;
+
+namespace TDSQASystemAPI.TestPackage
+{
+    /// <summary>
+    /// A partial class to augment the <code>PresentationsPresentation</code> with additional 
+    /// data.
+    /// </summary>
+    public partial class PresentationsPresentation
+    {
+        private readonly IDictionary<string, string> labelMap = new Dictionary<string, string>
+        {
+            { "ENU", "English" },
+            { "ESN", "Spanish" },
+            { "ENU-Braille", "Braille" }
+        };
+
+        /// <summary>
+        /// Get the correct display label based on this presentation's label or code.
+        /// </summary>
+        /// <remarks>
+        /// <para>
+        /// THIS METHOD SHOULD BE USED INSTEAD OF THE label PROPERTY.  The logic for this method comes from
+        /// the <code>getLabel()</code>method in /TDS_SupportTool/client/src/main/java/tds/testpackage/model/Presentation.java
+        /// </para>
+        /// <para>
+        /// There is no easy way to override a property in a partial class, so this method is used to 
+        /// fetch the correct label.
+        /// </para>
+        /// </remarks>
+        /// <returns>The correct display label</returns>
+        public string GetLabel()
+        {

```

Comment:
Having this method is a bit of a smell to me.  I tried creating a class that derived from `PresentationsPresentation` and overriding/overwriting the `label` property, but no luck.  Unfortunately, there doesn't seem to be a way to override/overwrite properties with partial classes.  There are partial _methods_, but those don't seem to work either.

I'd prefer not to modify the classes that were generated by `xsd.exe`, so I punted and created this `GetLabel()` method.